raiden/storage/sqlite.py

```diff
@@ -981,8 +981,8 @@ def get_raiden_events_payment_history_with_timestamps(
             partner_address=partner_address,
         )
         return [
-            TimestampedEvent(self.serializer.deserialize(event.wrapped_event), event.log_time)
-            for event in events
+            TimestampedEvent(self.serializer.deserialize(data), timestamp)
```

Comment:
If we use `List[Tuple[str, datetime]]` above we could just as well use `List[Tuple[Event, datetime]]` here and get rid of the TimestampedEvent type. That would be more consistent.